x-pack/plugins/code/server/utils/esclient_with_internal_request.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Server } from 'hapi';
+import { AnyObject, EsClient } from '../lib/esqueue';
+import { EsIndexClient } from './es_index_client';
+import { WithInternalRequest } from './with_internal_request';
+
+export class EsClientWithInternalRequest extends WithInternalRequest implements EsClient {
+  public readonly indices = new EsIndexClient(this);
```

Comment:
is this field used somewhere?

Reply:
this is the stub to execute all the index related APIs on Elasticsearch, e.g. index_worker to create indices.